src/python/MovingWindowFilter.i

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+%module movingwindowfilter
+%{
+#include "ignition/math/Export.hh"
+#include "ignition/math/MovingWindowFilter.hh"
+#include "ignition/math/Vector3.hh"
+%}
+
+%import "std_vector.i"
```

Comment:
Removing this line fixes the compilation error for me. Maybe it's a known issue, but I had to `touch python.i` for it to recompile this file.

Reply:
Let me take a look. It seemed to be working before merging this branch with ign-math6.

Reply:
You were right. That import was causing issues because of the syntaxis: it should be `include` instead of `import`

The truth is that that import wasn't necessary given that `std::vector` isn't being used in the API, so that import statement wasn't needed at all. I pushed the fix